packages/ui-core/src/components/color-scale/color-scale.component.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { colors as themeColors } from '@keen.io/colors';
+
+import { ColorModeType } from '../../types';
+
+import { ColorCard, ColorGradient } from './color-scale.styles';
+
+type Props = {
+  colors?: string[];
+  colorMode?: ColorModeType;
+};
+
+const ColorScale = ({ colors, colorMode = 'shades' }: Props) => {
+  const colorsArray =
+    colors && colors.length
+      ? colors.length > 1
+        ? colors
+        : ['white', colors[0]]
+      : ['white', themeColors.lightBlue[500]];
+
+  return (
+    <>
+      {colorMode === 'shades' || !colorsArray ? (
+        <ColorGradient
+          key={`${colorsArray[0]}-${colorsArray[1]}`}
```

Comment:
I believe that `key` property is not required here as we do not map a collection to components.